test/src/actions/ios/welcome.actions.js

```diff
@@ -25,6 +26,34 @@ class WelcomeActions extends AndroidWelcomeActions {
         }
         FilesHelper.editJsonByKey(walletData, "masterKey", masterKey);
     }
+
+    async addMasterKey() {
+        for (let i = 0; i < 24;) {
+            for (let j = 1; j <= 3; j++) {
+                await ActionHelper.setText(Pages.welcomePage.typeNemonic(j), importData.importWallet[i]);
+                await ActionHelper.pause(500);
+                if (await $(Pages.welcomePage.wordSuggestion).isDisplayed()) {
+                    await ActionHelper.click(Pages.welcomePage.wordSuggestion);
+                }
+                i++
+            }
+            await ActionHelper.click(Pages.welcomePage.rightNavigationArrow);
+        }
+    }
+
+    async setPin() {
+        await ActionHelper.pause(6000);
```

Comment:
instead of pause, we can use anymore the wait-helper methods which should be more efficient ;)